go/felix/rules/rule_defs.go

```diff
@@ -31,8 +31,8 @@ const (
 	ChainFilterForward = ChainNamePrefix + "-FORWARD"
 	ChainFilterOutput  = ChainNamePrefix + "-OUTPUT"
 
-	ChainFailsafeIn  = ChainNamePrefix + "-FAILSAFE-IN"
-	ChainFailsafeOut = ChainNamePrefix + "-FAILSAFE-OUT"
+	ChainFailsafeIn  = ChainNamePrefix + "-failsafe-in"
```

Comment:
Upper case is reserved for our versions of top-level chains i.e. the `FORWARD` chain jumps to `cali-FORWARD`.